addons/http_routing/models/ir_http.py

```diff
@@ -113,6 +113,14 @@ def unslug(s):
         return None, None
     return m.group(1), int(m.group(2))
 
+# TODO: remove in master
+def unslug_object_str(s):
+    """ From a string like 'blog.blog(2,)' to {'model': 'blog.blog', 'id': 2} """
+    m = re.search(r'(.+)\((\d+),(.*)\)', s)
+    if not m:
+        return None, None
+    return {'model': m.group(1), 'id': int(m.group(2))}
```

Comment:
I know we talked about using `unslug` but once you found out I was wrong because `blog.blog(` is actually not a slug... why did not know ask / make just a local patch in the controller?